srcpkgs/greetd/files/greetd/run

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
+
+exec greetd
```

Comment:
Do you think it would make sense to redirect stdeer?
```
exec 2>&1
```

Reply:
Only if greetd spews stuff to stderr instead of stdout, which I have no insight with.